### ISSUE TITLE
Strange issues with the Toggle Comment command

### ISSUE THREAD

**Issue by jklaiho** [clifford-github/sublime-ansible#40]
I'm seeing strange behaviour from Sublime Text (build 4152) Toggle Comment command and this package.

Assume the following YAML (line numbers added for referring to selection regions):

```
1  ---
2  - name: test
3    command: sleep 1
4
5  - block:
6      - name: test
7        command: sleep 1
```

Some test cases for toggling selected lines:

- All lines (1-7), toggling works.
- Lines 2-3, **does not work**
- Lines 5-7, works.
- Lines 6-7, works
- Lines 2-7, **does not work**

Individual line test cases:

- Cursor on line 2, no selection: **does not work**
- Cursor on any other line, no selection: works

If I set the syntax of the file as just YAML, both non-working cases with the Ansible syntax start working.

In addition, if I toggle comments ON for lines 2-3 or 2-7 while in YAML syntax, then switch back to the Ansible syntax, I can toggle comments OFF, but not back ON.

So, it seems that if the selection region starts with a top-level `- name: something` section, toggling comments ON fails. However, that same section inside a `- block` section toggles just fine.

**Comment by killua99** [comment]
Info:

Sublime Text version: 4152
Package version: 22.12.1 

This is the result:
<img width="359" alt="untitled_—_sublime-ansible" src="https://github.com/clifford-github/sublime-ansible/assets/457822/29d518e9-c066-4ff9-8832-b91a5b85aec1">

**Comment by killua99** [comment]
@jklaiho before I commit a new release tag, could you please download the master branch in your local, and git it a try.

**Comment by jklaiho** [comment]
@killua99 Yep, the master branch worked fine on all of the seven test cases I listed in my original issue!
